jme3-core/src/com/jme3/gde/core/assets/ProjectAssetManager.java

```diff
@@ -478,16 +524,19 @@ private List<String> collectDependenciesFilesWithSuffix(String suffix, List<Stri
             while (classPathItemsIter.hasNext()) {
                 ClassPathItem classPathItem = classPathItemsIter.next();
                 FileObject jarFile = classPathItem.object;
-
-                Enumeration<FileObject> jarEntry = (Enumeration<FileObject>) jarFile.getChildren(true);
-                while (jarEntry.hasMoreElements()) {
-                    FileObject jarEntryAsset = jarEntry.nextElement();
-                    if (jarEntryAsset.getExt().equalsIgnoreCase(suffix)) {
-                        if (!jarEntryAsset.getPath().startsWith("/")) {
-                            list.add(jarEntryAsset.getPath());
+                if(FileUtil.isArchiveFile(jarFile)) {
```

Comment:
It's this change that causes MatDefs not to show in ant projects

Reply:
Hye @neph1 ! Thanks for testing :+1: 

This is exactly the bit I thought might break Ant :cry: I can't remember exactly, but from memory I think from the NB Ant Project API we get the dependent jars with `jar://` URLs, but from Gradle Project API we get back `file://` URLs.

Basically, from Ant projects we explicitly knew they were Jar files, whereas in Gradle I'm just treating them here as a regular .zip file and trying to add everything in it for classloading.

I should have a bit of free time this weekend to look at this :+1:

Reply:
@neph1 - try now please 😺 

p.s. please squash if/when you merge this is - I tidied up a couple of typos and the fact I forgot to switch the logger back to being less 'chatty'